src/asmcnc/apps/drywall_cutter_app/screen_config_filesaver.py

```diff
@@ -0,0 +1,483 @@
+                    self.background_color = hex('#FFFFFF00')
+                on_press:
+                    self.background_color = hex('#FFFFFFFF')
+                BoxLayout:
+                    padding: 25
+                    size: self.parent.size
+                    pos: self.parent.pos
+                    
+            Button:
+                disabled: True
+                size_hint_x: 1
+                background_color: hex('#FFFFFF00')
+                on_release: 
+                    self.background_color = hex('#FFFFFF00')
+                on_press:
+                    root.get_FTP_files()
+                    root.refresh_filechooser() 
+                    self.background_color = hex('#FFFFFFFF')
+                BoxLayout:
+                    padding: 25
+                    size: self.parent.size
+                    pos: self.parent.pos
+                    
+            Button:
+                id: delete_selected_button
+                disabled: True
+                size_hint_x: 1
+                background_color: hex('#FFFFFF00')
+                on_release: 
+                    root.delete_popup(file_selection = filechooser.selection[0])
+                    self.background_color = hex('#FFFFFF00')
+                on_press:
+                    self.background_color = hex('#FFFFFFFF')
+                BoxLayout:
+                    padding: 25
+                    size: self.parent.size
+                    pos: self.parent.pos
+            Button:
+                id: delete_all_button
+                disabled: True
+                size_hint_x: 1
+                background_color: hex('#FFFFFF00')
+                on_release: 
+                    self.background_color = hex('#FFFFFF00')
+                on_press:
+                    root.delete_popup(file_selection = 'all')
+                    self.background_color = hex('#FFFFFFFF')
+                BoxLayout:
+                    padding: 25
+                    size: self.parent.size
+                    pos: self.parent.pos
+            Button:
+                disabled: False
+                size_hint_x: 1
+                background_color: hex('#FFFFFF00')
+                on_release: 
+                    root.quit_to_home()
+                    self.background_color = hex('#FFFFFF00')
+                on_press:
+                    self.background_color = hex('#FFFFFFFF')
+                BoxLayout:
+                    padding: 25
+                    size: self.parent.size
+                    pos: self.parent.pos
+                    Image:
+                        id: image_cancel
+                        source: "./asmcnc/skavaUI/img/file_select_cancel.png"
+                        center_x: self.parent.center_x
+                        y: self.parent.y
+                        size: self.parent.width, self.parent.height
+                        allow_stretch: True 
+            Button:
+                id: load_button
```

Comment:
save_button - have a check through and make sure it's clear that things here are saving rather than loading :)